### ISSUE TITLE
[React.js] Webpack < 5 used to include polyfills for node.js

### ISSUE THREAD

**Issue by JamesFox16** [terra-money/wallet-provider#54]
The latest version of `create-react-app` uses `react-scripts: 5.0.0` which causes issues with dependencies used in the `wallet-provider` package. 

When I run using `react-scripts: 5.0.0`, I get the following
```
ERROR in ./node_modules/cipher-base/index.js 3:16-43

Module not found: Error: Can't resolve 'stream' in '.../node_modules/cipher-base'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
	- install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "stream": false }
```

### To Reproduce
1. Clone example project.
```
npx terra-templates get wallet-provider:create-react-app your-app-name
```
2. Change `react-scripts` from version 4.x to version 5.x install and run.
```
yarn add --exact react-scripts@5.0.0
yarn install
yarn start
```

From what I saw, the other dependencies affected by this upgrade are `readable-stream` and `safe-buffer`. For now, the app and packages will run by downgrading to `react-scripts: 4.0.3`.

**Comment by iamssen** [comment]
Yes, eventually, this time has come. 😅 This problem arises because terra.js relies on node polyfill below webpack 4.0.

You can refer to this link for the polyfill configuration required to run terra.js on webpack 5.0. 

<https://github.com/terra-money/terra-web-extensions/blob/main/apps/scripts/webpackConfigs.ts#L33>

I will update the create-react-app template based on webpack 5.0 within this week. 

I don't want to use rewired on the basic template, but I think I have to do it.
